src/management/api/analytics/logs/logging-configuration.controller.ts

```diff
@@ -33,43 +38,118 @@ class ApiLoggingConfigurationController {
     private $stateParams,
     private $rootScope,
     private $scope,
-    private Constants
+    private Constants,
+    private spelGrammar: any,
```

Comment:
Sad to have an `any` here but it looks like the type isn't available yet and is quite 🤯 . Next time